charts/gardener-dashboard/values.yaml

```diff
@@ -101,8 +101,11 @@ oidc:
   #  # clientSecret is the public client secret use by kubelogin and all users
   #  clientSecret: ~
 
-frontendConfig:
+# frontendConfigPublic is the configuration that is accessible without authentication
+frontendConfigPublic:
```

Comment:
I would keep the `landingPageUrl` in `frontendConfig` and handle what is public in the tmeplate for now.

Reply:
✅